spec/lib/metasploit/framework/login_scanner/http_spec.rb

```diff
@@ -30,4 +35,160 @@
     end
   end
 
+  describe '#set_http_trace_proc' do
+    let(:sample_request) {
+      "GET / HTTP/1.1\nHost: www.google.com"
+    }
+
+    let(:sample_response) {
+      res = Rex::Proto::Http::Response.new(302,'Found')
+      allow(res).to receive(:body).and_return("Location: https://www.google.com/?gws_rd=ssl")
+      res
+    }
+
+    let(:expected_output) {
```

Comment:
I think this could be renamed as this isn't very descriptive. What is expected? That could depend on the context. A better name would be something about it being a full request and response.